src/libs/ajax/workspaces/Workspaces.ts

```diff
@@ -173,7 +174,12 @@ export const Workspaces = (signal?: AbortSignal) => ({
 
       checkBucketAccess: GoogleStorage(signal).checkBucketAccess,
 
-      checkBucketLocation: GoogleStorage(signal).checkBucketLocation,
+      checkBucketLocation: async (userProject?) => {
```

Comment:
Can we remove `checkBucketLocation` from `GoogleStorage` or is that still getting used somewhere?

Reply:
It is not being used, I can remove it.